lib/fastlane/actions/resign.rb

```diff
@@ -33,8 +47,12 @@ def self.available_options
                                        env_name: "FL_RESIGN_PROVISIONING_PROFILE",
                                        description: "Path to your provisioning_profile. Optional if you use `sigh`",
                                        default_value: Actions.lane_context[SharedValues::SIGH_PROFILE_PATH],
+                                       is_string: false,
                                        verify_block: proc do |value|
-                                         raise "No provisioning_profile file given or found, pass using `provisioning_profile: 'path/app.mobileprovision'`".red unless File.exist?(value)
+                                         files = value.kind_of?(Enumerable) ? value.map { |fst, snd| snd || fst } : [value]
```

Comment:
We could write something like this:
```ruby
files = case value
when Enumerable; value.map { |a, b| b || a }
else [value]
end
```
I know the `map` is a little bit confusing. But It's the simplest to support both hashes and arrays of strings and also arrays of strings and tuples. For example, we could imagine passing the following value:
```ruby
["app.mobileprovision", ["com.example.app.extension", "ext.mobileprovision"]]
```
That would be translated as:
```
-p app.mobileprovision -p com.example.app.extension=ext.mobileprovision
```
Or we can assume that nobody needs that. In which case, I propose this:
```ruby
files = case value
when Hash; value.values
when Enumerable; value
else [value]
end
```